packages/graph/src/graph2/graph.ts

```diff
@@ -209,6 +200,22 @@ export class Graph2 extends SVGZoomWidget {
         this.zoomToFitLimit(1);
     }
 
+    resolveAnnoEventNode(): undefined | IVertex4Annotation {
+        const d3evt = d3Event();
+        const eventPath = d3evt?.sourceEvent?.path ?? d3evt.path;
+        const anno = eventPath.find(n => {
+            if(typeof n.hasAttribute === "function") {
+                return !!n.hasAttribute("data-anno");
```

Comment:
Is there any need for the `!!`?

Reply:
It's really just a visual reminder that the code is looking for truthiness.

Reply:
hasAttribute already returns a boolean?